src/Agent/NewRelic/Agent/Core/AgentManager.cs

```diff
@@ -282,7 +282,7 @@ private void LogInitialized()
                     }
                 }
 
-                Log.Debug($".NET Runtime Version: {RuntimeInformation.FrameworkDescription}");
+                Log.Info($".NET Runtime Version: {RuntimeInformation.FrameworkDescription}");
```

Comment:
I don't think that this log line will give you the information that you want for all of the versions of .net (especially .net framework). You might want to look at the changes in #1852 which call out when a framework is unsupported and is already at a higher log level.